ui/ResultViewer.ui

```diff
@@ -34,7 +34,7 @@
          </font>
         </property>
         <property name="toolTip">
-         <string>Saves results of this evaluation.</string>
+         <string>Saves results of this evaluation. (Ctrl+S)</string>
```

Comment:
I am not UX guy, but I think we could have Save results option in main menu and it could be grayed out with no results are available.